frontend/src/utils/formatters.ts

```diff
@@ -44,14 +45,5 @@ export function toKebabCase(str: string | null) {
  * @returns {number} A converted string representation of `bytes`
  */
 export function formatSize(bytes: number) {
-  if (bytes === 0) {
-    return '0 B';
-  }
-
-  const k = 1000,
-    dm = 3,
-    sizes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
-    i = Math.floor(Math.log(bytes) / Math.log(k));
-
-  return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+  return filesize(bytes);
```

Comment:
If this is just a one-liner now, do we need the formatSize function as a wrapper? I'd recommend instead that we back this up a layer or two and just directly include filesize import directly where we need it. Glad the library works though!